aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -357,6 +423,28 @@ module AptosFramework::Stake {
         );
     }
 
+    /// Allows an owner to change the delegated voter of the stake pool.
+    public(script) fun set_delegated_voter(
+        account: &signer,
+        new_delegated_voter: address,
+    ) acquires OwnerCapability, StakePool {
+        let account_addr = Signer::address_of(account);
+        let ownership_cap = borrow_global<OwnerCapability>(account_addr);
+        set_delegated_voter_with_cap(account_addr, ownership_cap, new_delegated_voter);
+    }
+
+    /// Allows an owner to change the delegated voter of the stake pool.
+    public(script) fun set_delegated_voter_with_cap(
+        pool_address: address,
+        owner_cap: &OwnerCapability,
```

Comment:
This is a security issue and conflicts with `validate_combine_signer_and_txn_args`:
```
   for ty in func.parameters[signer_param_cnt..].iter() {
       if !AptosVM::is_valid_for_constant_type(ty) || matches!(ty, Type::Signer) {
           return Err(VMStatus::Error(StatusCode::INVALID_MAIN_FUNCTION_SIGNATURE));
       }
   }
```
@movekevin

Reply:
I'm not sure why this would be a security issue but yes this should be public since public(script) cannot construct OwnerCapability anyway. Thanks for the catch. I'll fix that in a followup PR :)